ecmascript/parser/tests/typescript/function/predicate-types/input.ts.json

```diff
@@ -222,6 +224,113 @@
           }
         }
       }
+    },
+    {
+      "type": "FunctionDeclaration",
+      "identifier": {
+        "type": "Identifier",
+        "span": {
+          "start": 81,
+          "end": 82,
+          "ctxt": 0
+        },
+        "value": "g",
+        "typeAnnotation": null,
+        "optional": false
+      },
+      "declare": false,
+      "params": [
+        {
+          "type": "Identifier",
+          "span": {
+            "start": 83,
+            "end": 89,
+            "ctxt": 0
+          },
+          "value": "x",
+          "typeAnnotation": {
+            "type": "TsTypeAnnotation",
+            "span": {
+              "start": 84,
+              "end": 89,
+              "ctxt": 0
+            },
+            "typeAnnotation": {
+              "type": "TsKeywordType",
+              "span": {
+                "start": 86,
+                "end": 89,
+                "ctxt": 0
+              },
+              "kind": "any"
+            }
+          },
+          "optional": false
+        }
+      ],
+      "decorators": [],
+      "span": {
+        "start": 72,
+        "end": 115,
+        "ctxt": 0
+      },
+      "body": {
+        "type": "BlockStatement",
+        "span": {
+          "start": 113,
+          "end": 115,
+          "ctxt": 0
+        },
+        "stmts": []
+      },
+      "generator": false,
+      "async": false,
+      "typeParameters": null,
+      "returnType": {
+        "type": "TsTypeAnnotation",
+        "span": {
+          "start": 90,
+          "end": 112,
+          "ctxt": 0
+        },
+        "typeAnnotation": {
+          "type": "TsTypePredicate",
+          "span": {
+            "start": 90,
```

Comment:
As pointed out above, this should be 92, but fixing it causes too many other changes for this PR.